### PR TITLE
Add our podcast to the listed projects

### LINKED ISSUE #19: List Let's talk about public code in our projects
Our podcast can be called a project/resource, and we should add it here. (Idea from @ElenaFdR in our chat.)

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #19 

-----
[View rendered index.md](https://github.com/Ainali/projects/blob/ja-add-podcast/index.md)

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -20,6 +20,11 @@ An interactive game exploring governance of a public codebase. It helps particip
 
 The game is also useful as a tool for visualizing how a current governance model is set up or could be changed.
 
+### [Let's talk about public code](https://podcast.publiccode.net)
+
+A podcast where we interview people working with public code codebases.
```
Comment (ElenaFdR) [inline index.md:25]: The guest selection criteria on About are different: https://about.publiccode.net/activities/live-streaming/

Comment (Ainali) [reply index.md:25]: Good catch! How about this?

`A podcast where we interview people with interesting perspectives on public code.`

Comment (ElenaFdR) [reply index.md:25]: Super!

**Review by ElenaFdR — APPROVED** [review]
Ready to go!

**Pull request merged**